cacheops/redis.py

```diff
@@ -58,7 +100,10 @@ def __setattr__(self, name, value):
 try:
     redis_conf = settings.CACHEOPS_REDIS
     redis_replica_conf = settings.CACHEOPS_REDIS_REPLICA
-    redis_replica = redis.StrictRedis(**redis_replica_conf)
+
+    class SafeReplicaRedis(redis.StrictRedis):
```

Comment:
Before PR degrade_on_failure didn't work for replica?

BTW, ReplicaProxyRedis and SafeReplicaRedis have different parents. Is it ok?